core/src/main/kotlin/com/rarible/protocol/union/core/util/LogUtils.kt

```
@@ -36,10 +33,11 @@ object LogUtils {
         router: BlockchainRouter<ItemService>,
         block: suspend CoroutineScope.() -> T
     ): T {
-        val collection = if (itemId.blockchain == BlockchainDto.SOLANA) {
-            getSolanaCollection(itemId, router)
-        } else {
-            getCollection(itemId)
+        val collection = try {
+            router.getService(itemId.blockchain).getItemCollectionId(itemId.value) ?: ""
+        } catch (e: Exception) {
+            // should never happen
```

maybe add any log message here? Empty catch block bothers my eyes )